magni_bringup/config/default_robot.yaml

```diff
@@ -38,6 +38,8 @@ ubiquity_motor:
   wheel_type: "standard"
   wheel_gear_ratio: 4.294
   fw_max_pwm: 325
+  fw_max_speed_fwd: 100
```

Comment:
Firmware currently uses 104 and -104 so we should remain consistent.

Reply:
so this should be changed to:
```
  fw_max_speed_fwd: 104
  fw_max_speed_rev: -104
```
?

Reply:
Yes.  This is sort of a nitpik and I think 100 would be ok but if we keep it 104 absolute value numbers there will be less confusion later.   Not that it matters but this is how that comes about:

80  is 1 meter per sec and enc tic rate of 66hz. Wheels are 0.63M/rev and 80 is 0.65 sec/rev
104 is just over 1.25 meter per sec.  Same units for 3 or 6 state code to avoid confusion

Reply:
Mkay, done. Can you recheck if its ok?